README.md

```diff
@@ -1,5 +1,54 @@
+[![MoreCompany Discord](https://img.shields.io/badge/MoreCompany%20Discord-5865F2?style=for-the-badge)](https://discord.gg/cKa6sPBFZ9) [![GitHub Issue Tracker](https://img.shields.io/badge/Issue%20Tracker-2dba4e?style=for-the-badge)](https://github.com/notnotnotswipez/MoreCompany/issues)
 # MoreCompany
-Hello,
-this mod is not affiliated with BiggerLobby.
 
-Although the license of the mod is MIT, it would be really silly to take all of my patches and put them in another mod or just reupload a fork. I can't do anything legally about it but yknow, please?
+MoreCompany expands Lethal Company's max player count for larger player lobbies and adds new cosmetics, all in a stable experience. Some UI elements may not fully support more than 8 players. Available to download from [Thunderstore](https://thunderstore.io/c/lethal-company/p/notnotnotswipez/MoreCompany/) or view the source code on [GitHub](https://github.com/notnotnotswipez/MoreCompany).
+
+## Release Trailer
+[![](http://img.youtube.com/vi/yHxU4YrA8Ag/0.jpg)](http://www.youtube.com/watch?v=yHxU4YrA8Ag "MoreCompany Launch Trailer on YouTube")
+
+## Features
+- Cosmetics for players to choose in menu and wear in-game, and the ability to create your own (just check out the [guide](https://github.com/notnotnotswipez/MoreCompany/wiki/Cosmetic-Creation))
+- Scavenge results modified to fit 8 players and will show their status at the end of the round.
+- Spectators screen properly fits more than 4 players, and will not go offscreen.
+- The player volume menu accounts for more than 4 players, by adding a scrollbar to the UI.
+
+## Issues
+If you encounter any issues with the mod, please report them on the [MoreCompany Discord](https://discord.gg/cKa6sPBFZ9) or via the [GitHub Issue Tracker](https://github.com/notnotnotswipez/MoreCompany/issues). Please ensure you check for known issues on the [GitHub Issue Tracker](https://github.com/notnotnotswipez/MoreCompany/issues) or [`CHANGELOG.md`](https://github.com/notnotnotswipez/MoreCompany/blob/master/CHANGELOG.md) BEFORE reporting.
+
+# Why choose MoreCompany?
+Although there is another mod that does a very similar thing, it has a few stability issues regarding the functionality of the game. In the development of MoreCompany, I had stability and polish in mind, I wanted to make a relatively frictionless experience regarding expanded player counts.
```

Comment:
```suggestion
In the development of MoreCompany, I had stability and polish in mind, I wanted to make a relatively frictionless experience regarding expanded player counts.
```
Since betterlobby is deprecated there isn't rly much need to mention the "another mod" part